Loop/Public/SwiftUI/LoopBinding.swift

```diff
@@ -0,0 +1,49 @@
+import SwiftUI
+import Combine
+import ReactiveSwift
+
+@available(macOS 10.15, iOS 13.0, tvOS 13.0, watchOS 6.0, *)
+@propertyWrapper
+public struct LoopBinding<State, Event>: DynamicProperty {
+    @ObservedObject
+    private var subscription: SwiftUISubscription<State, Event>
+
+    private let loop: Loop<State, Event>
+
+    @inlinable
+    public var wrappedValue: State {
+        acknowledgedState
+    }
+
+    public var projectedValue: LoopBinding<State, Event> {
+        self
+    }
+
+    @usableFromInline
+    internal var acknowledgedState: State!
+
+    public init(_ loop: Loop<State, Event>) {
+        // The subscription can be copied without restrictions.
+        self.subscription = SwiftUISubscription()
```

Comment:
I feel like we can get rid of a lot implicitly unwrapped optionals if we inject loop into subscription and attach it internally by calling a method? or dunno rename it to `start()`

Reply:
Doesn't work with `@EnvironmentLoop`.